src/sql/workbench/contrib/dashboard/browser/widgets/insights/insightsWidget.css

```diff
@@ -0,0 +1,15 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+.insights-widget-last-updated {
+	font-size: 10px;
+	margin-left: 5px;
+	color: #484644;
+}
+
+.vs-dark .lastUpdated,
```

Comment:
Is this class correct? Shouldn't it be `.insights-widget-last-updated`?

(that being said - as Alan mentioned it would be better to put these in the color registry)

Reply:
removed this after moving colors